cli/util/options.d.ts

```diff
@@ -3,7 +3,7 @@ export interface OptionDescription {
   /** Textual description. */
   description?: string | string[],
   /** Data type. One of (b)oolean [default], (i)nteger, (f)loat or (s)tring. Uppercase means multiple values. */
-  type?: "b" | "i" | "f" | "s", "I", "F", "S",
+  type?: "b" | "i" | "f" | "s" | "I" | "F" | "S",
```

Comment:
nice find! did that even work before?

Reply:
Nut sure it was ever used.  Since it was just the type definition file and no one tried to use the type with a typescript project.